doc/source/whatsnew/v0.25.0.rst

```diff
@@ -743,6 +743,7 @@ Other
 - Removed unused C functions from vendored UltraJSON implementation (:issue:`26198`)
 - Bug in :func:`factorize` when passing an ``ExtensionArray`` with a custom ``na_sentinel`` (:issue:`25696`).
 - Allow :class:`Index` and :class:`RangeIndex` to be passed to numpy ``min`` and ``max`` functions (:issue:`26125`)
+- Bug fixed in :meth:`Interval.overlaps` and :meth:`IntervalArray.overlaps` handling of empty intervals (:issue:`26893`)
```

Comment:
Technically this is also fixed in `IntervalIndex.overlaps`, since it shares the `IntervalArray.overlaps` code.

Reply:
fixed.